OpenIPSL/Electrical/ThreePhase/package.mo

```diff
@@ -17,5 +17,5 @@ annotation (
 </tr>
 </table>
 </html>"),
-  version="1");
+  conversion(noneFromVersion="1"));
```

Comment:
@dietmarw Should I delete this command? I don't see why we need to maintain this

Reply:
Yes this has no function in a non-toplevel package.mo